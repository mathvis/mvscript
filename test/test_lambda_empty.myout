Expr (LambdaFunc [] (Just (Block FunctionBlock [])))
