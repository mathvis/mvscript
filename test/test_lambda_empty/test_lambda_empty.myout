Expr (LambdaFunc [] (Just (Block (FunctionBlock void) [])))
