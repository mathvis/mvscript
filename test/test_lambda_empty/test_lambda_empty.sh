#!/bin/bash
helix test_lambda_empty.mvs test_lambda_empty.out --vsplit
