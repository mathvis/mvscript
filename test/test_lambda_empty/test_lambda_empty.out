Expr (LambdaFunc [] (Just (Block (FunctionBlock VoidT) [])))
